### DISCUSSION TITLE
v7.2.1 is not in npmjs

### DISCUSSION THREAD

**Discussion by gigaga** [mobxjs/mobx#3143] [General]
Hi,

v7.2.1 doesn't seem to be in npmjs. Indeed, https://www.npmjs.com/package/mobx-react/v/7.2.1 --> 404.

Regards,

**Comment by iChenLei** [comment]
We mentioned it, see here https://github.com/mobxjs/mobx/pull/3141#issuecomment-941123803

**Comment by mweststrate** [comment]
I'll try to take a look tonight
